### PR TITLE
Flip commit type

### PR DESCRIPTION
**Pull request by dweberdev**
Add Flip UI to commit modal

Will add flip and summary logic with #432 and #434 

Screen Shot 2022-02-04 at 1 45 52 PM
Screen Shot 2022-02-04 at 1 46 13 PM

### CODE REVIEW

File: components/General/TWModal/index.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ export const TWModal: React.FC<TWModalProps> = ({ open, onClose, size = 'default
     return (
         <Transition.Root show={open} as={Fragment}>
             <Dialog as="div" className="fixed z-10 inset-0 overflow-y-auto" onClose={() => onClose()}>
-                <div className="flex items-end justify-center min-h-screen pt-4 px-4 pb-20 text-center sm:block sm:p-0">
+                <div className="flex items-end justify-center min-h-screen pt-4 pb-20 text-center sm:block sm:p-0">
```
Comment (dospore) [inline components/General/TWModal/index.tsx:24]: do you mind adding a min-width 611px to this

File: libs/utils/tradeHistoryAPI.ts [hunk 2]
```diff
@@ -44,7 +44,7 @@ export type TradeHistory = {
 const fetchTradeHistory: (params: {
     network?: NetworkType;
     account: string;
-    type: 'mint' | 'burn';
+    type: string;
```
Comment (dospore) [inline libs/utils/tradeHistoryAPI.ts:47]: 'mint' | 'burn' | 'flip' ??

**Review by dospore — COMMENTED** [review]
lgtm couple of comments but happy to approve

File: components/General/TWModal/index.tsx (hunk 1 above)
Comment (dweberdev) [reply components/General/TWModal/index.tsx]: Added a styled component with a media query min-width: 611px

I wanted to do the same for Dialog.Overlay but I am having a hard time with typescript, styled components and headlessUI compound component structure. TS is throwing some crazy errors when passing props.

Type '{ isOverlay: boolean; }' is not assignable to type 'IntrinsicAttributes & Props<"div", OverlayRenderPropArg, OverlayPropsWeControl>'.

**Review by Steven-Yulong-Yan — APPROVED** [review]

**Pull request merged**